build-amp-image.sh

```diff
@@ -78,6 +78,11 @@ if [ $? -ne 0 ]; then
   echo "failed"
   exit 1
 fi
+$DOCKER cp amp-builder:/go/bin/amp-haproxy ./images/haproxy/amp-haproxy.alpine >&2
```

Comment:
no, for sure. I didn't finish this part considering you are refactoring all the make files